backend/tracim_backend/models/tracim_session.py

```diff
@@ -0,0 +1,24 @@
+from contextlib import contextmanager
+
+from sqlalchemy.orm import Session
+
+
+class TracimSession(Session):
+    """
+    Subclass of Sqlalchemy session to add tracim specific stuff
+    """
+
+    def __init__(self, **kwargs):
+        Session.__init__(self, **kwargs)
+        self.allow_revision_deletion = False
+
+
+@contextmanager
+def unsafe_tracim_session(session: TracimSession):
+    """
+    allow to bypass protection on tracim revisions
+    """
+    original_allow_revision_deletion = session.allow_revision_deletion
+    session.allow_revision_deletion = True
```

Comment:
I'm not found of this pattern which uses a statefull class instead of building a dedicated object. I'd like to talk about this tomorrow moring in order to be sure there is no better way to do it